### PR TITLE
Trigger on file modifications

### PR DESCRIPTION
**Pull request by sol**

### CODE REVIEW

**Comment by sol** [comment]
Based on #1.

File: src/Run.hs [hunk 1]
```diff
@@ -1,17 +1,57 @@
...
+processEvents chan interpreter = do
+  t <- getCurrentTime
+  emitEvent chan
+  go t
+  where
+    go t0 = do
+      event <- readChan chan
+      case event of
+        Done -> return ()
+        Event t | t0 < t -> do
```
Comment (soenkehahn) [inline src/Run.hs]: I found this hard to understand. Wouldn't it be easier to empty all events from the chan after the `threadDelay`?

**Comment by soenkehahn** [comment]
Apart from the comment LGTM.

File: src/Run.hs (hunk 1 above)
Comment (soenkehahn) [inline src/Run.hs]: Hmm, but I see that emtying the channel is not that straight-forward. `isEmptyChan` is deprecated.

**Pull request merged**